apps/storybook-e2e/src/e2e/components/next/form/checkbox/vl-checkbox.stories.cy.ts

```diff
@@ -0,0 +1,20 @@
+const checkboxNextUrl =
+    'http://localhost:8080/iframe.html?args=&id=components-next-form-checkbox--checkbox-default&viewMode=story';
+const checkboxNextSwitchUrl =
+    'http://localhost:8080/iframe.html?args=&id=components-next-form-checkbox--checkbox-switch&viewMode=story';
+
+describe('story vl-checkbox-next default', () => {
+    it('should render story', () => {
+        cy.visit(checkboxNextUrl);
+
+        cy.get('vl-checkbox-next');
+    });
+});
+
```

Comment:
Zelfs als de component niet geregistreerd is en niet werkt gaat deze test lukken omdat de tag aanwezig is. Ik zou net iets meer willen testen: heeft hij de juiste style (css in aanwezig) en is het een volwaardige web-component (ik weet ook niet hoe, misschien verifieren of er een shadow-dom is?).

Reply:
@ksvlaanderen lijkt ok voor mij; dit impliceert dat we dit dan ook gaan uitvoeren voor alle cypress component tests (vanaf nu) wat denk je hiervan?

Reply:
Klinkt goed

Reply:
heb test toegevoegd die nu gerenderde style aftest + of shadow dom aanwezig is